scripts/approve_embargo_terminations.py

```diff
@@ -62,8 +62,14 @@ def main():
         else:
             count += 1
             logger.info('Ending the Embargo ({0}) of Registration ({1}) early. Making the registration and all of its children public now.'.format(embargo._id, registration._id))
+            # Manually marking EmbargoTerminationApproval as 'approved' instead of calling _on_approve() hook
+            # _on_approve hook would require that all users had given their approval.
+            request.state = models.Sanction.APPROVED
```

Comment:
No, in my opinion, this would mix up the current meanings of  `APPROVED` and `COMPLETED` on Sanctions. Furthermore, I'm leaning against a separate `AUTO_APPROVED` state, as this would require a larger overhaul because we check the specific value of this field in places - it is required to be `APPROVED` to cause certain behavior.  The general discussion to track person approval versus machine approval is going to be a larger one.

More ramblings as I think through this:
- `APPROVED`, `UNAPPROVED`, and `REJECTED` strings seem to be current reviews-like states for Sanctions.  
- My understanding is that `APPROVED` is, okay, we've all signed off on this thing (or it was done automatically), and `COMPLETED` means this is no longer a thing.  
- `COMPLETED` seems to be separate from these reviews states, solely for Embargoes.
-  Other cron jobs that automatically approve Sanctions generally mark them as `APPROVED`.
-  We don't want to have a blanket rule of cron jobs marking things `APPROVED`, as this would mess up Embargoes. Cron job goes to automatically approve embargo - marking it as COMPLETE would make it public, rather than marking it as APPROVED and allowing the embargo to exist.
- Many things for Sanctions rely on specific string matching - certain API fields
- This field isn't just for internal record keeping purposes, these different states correspond to what we return in the API and also handle how a sanction-link is processed.  For example, if your EmbargoTerminationApproval was already approved, and you navigated to an old link, if your sanction was `APPROVED`, we strip the query params and redirect you.
- We're tracking human approval versus machine approval on logs - although we have a PLAT ticket where there's one case where this isn't accurate.

Reply:
Ok, fair enough. Let's not change how sanctions work just yet :)